src/components/issue-list-item.component.js

```diff
@@ -13,16 +14,28 @@ type Props = {
   locale: string,
 };
 
+const Container = styled.View`
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: center;
+  padding-right: 10;
+  padding: 5;
+  border-bottom-width: 1;
+  border-bottom-color: ${colors.greyLight};
+`;
+
+const CommentsContainer = styled.View`
+  flex-direction: row;
+  padding-top: 13;
+`;
+
+const Comments = styled.Text`
+  padding-left: 5;
+  font-size: ${normalize(12)}
+  color: ${colors.greyBlue}
+`;
+
 const styles = StyleSheet.create({
-  container: {
-    flexDirection: 'row',
-    alignItems: 'flex-start',
-    justifyContent: 'center',
-    paddingRight: 10,
-    paddingVertical: 5,
-    borderBottomWidth: 1,
-    borderBottomColor: colors.greyLight,
-  },
   closedIssue: {
```

Comment:
I think it can be removed, because nowhere uses it.

Reply:
The closeIssue style is used in Line 70.

Reply:
Oh yes, I forget to expand those collapsed codes.